src/components/ApeAvatar/ApeAvatar.tsx

```diff
@@ -19,7 +19,9 @@ export const ApeAvatar = ({
 }) => {
   // TODO: simplify so all: <ApeAvatar path={getAvatarPath(p?.avatar)} />
   const p = profile ?? user?.profile;
-  const avatarPath = getAvatarPathWithFallback(p?.avatar, user?.name);
+  let avatar = p?.avatar;
+  if (avatar === '') avatar = user?.profile?.avatar;
```

Comment:
```suggestion
  const avatar = p?.avatar || user?.profile?.avatar;
```

Reply:
this is clever, thanks! I was falling asleep 😪